wcvendors/wcvendors
# Fixed: Media not assign to vendor

### All Submissions:






### Changes proposed in this Pull Request:



Closes #790 .

### How to test the changes in this Pull Request:

1. Edit a product with a media that was uploaded by other user.
2. Assign the product to a vendor and click on the checkbox - Assign media to vendor
3. Check the end of the vendor where the product was assigned to

### Other information:
